web_tool_script_3.R

```diff
@@ -216,10 +216,10 @@ if (file.exists(file.path(proc_input_path, portfolio_name, "overview_portfolio.r
 }
 
 
-indicies_equity_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_equity_portfolio.rda"))
-indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_bonds_portfolio.rda"))
-peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_equity_portfolio.rda"))
-peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "0_Indices_bonds_portfolio.rda"))
+indicies_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
+indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
+peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
+peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
```

Comment:
Can we follow up with a PR to save these files with the extension .rds?

Reply:
I've been thinking about a "data prep script", that does some of the tasks done when "new_data == TRUE". this could also include changing these file formats, and doing some additional renaming.

Reply:
@maurolepore maybe create issues for these follow-up PR requests rather than comments that will probably be forgotten once this is merged?